lib/heuristics/scheduling_heuristic.rb

```diff
@@ -28,6 +28,8 @@ class Scheduling
     include SchedulingEndPhase
 
     def initialize(vrp, job = nil)
+      @wrapper = OptimizerWrapper.config[:services][:ortools]
```

Comment:
Why ortools is used instead of another solver?

Reply:
It was me who insisted on this. In fact it doesn't matter which solver is used because the `@wrapper` variable is used to call functions from the parent class of the solvers -- i.e., `Wrapper` -- since heuristic doesn't have a copy of the `Wrapper` object the options were to create a new `Wrapper` object here or use the existing objects.

Reply:
I think we can use/instantiate a base class with necessary methods to work

Reply:
Yeah that would be more appropriate. I don't think I know how to that in Ruby. Cheers Fabien!